docs/spark/tutorials/get-started.md

```diff
@@ -27,17 +27,17 @@ Before you begin writing your app, you need to set up some prerequisite dependen
 
 To start building .NET apps, you need to download and install the .NET SDK (Software Development Kit).
 
-Download and install the [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0). Installing the SDK adds the `dotnet` toolchain to your PATH.
+Download and install the [.NET Core SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1). Installing the SDK adds the `dotnet` toolchain to your PATH.
 
-Once you've installed the .NET Core SDK, open a new command prompt and run `dotnet`.
+Once you've installed the .NET Core SDK, open a new command prompt or terminal and run `dotnet`.
 
-If the command runs and prints out information about how to use dotnet, can move to the next step. If you receive a `'dotnet' is not recognized as an internal or external command` error, make sure you opened a **new** command prompt before running the command.
+If the command runs and prints out information about how to use dotnet, can move to the next step. If you receive a `'dotnet' is not recognized as an internal or external command` error, make sure you opened a **new** command prompt or terminal before running the command.
 
 ### 2. Install Java
 
-Install [Java 8.1](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+Install [Java 8.1](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) for MacOS and Windows, or [OpenJDK 8](https://openjdk.java.net/install/) for Ubuntu.
 
-Select the appropriate version for your operating system. For example, select **jdk-8u201-windows-x64.exe** for a Windows x64 machine. Then, use the command `java` to verify the installation.
+Select the appropriate version for your operating system. For example, select **jdk-8u201-windows-x64.exe** for a Windows x64 machine or **jdk-8u231-macosx-x64.dmg** for MacOS. Then, use the command `java` to verify the installation.
 
 ![Java Download](https://dotnet.microsoft.com/static/images/java-jdk-downloads-windows.png?v=6BbJHoNyDO-PyYVciImr5wzh2AW_YHNcyb3p093AwPA)
```

Comment:
Since we are showing the example of how to download JDK for windows and MacOS, do we also want to include instructions of OpenJDK for Ubuntu?

Reply:
The link for OpenJDK has instructions, but the link for Java 8.1 is just the releases with no instructions.

Reply:
Got it! Sounds good.